### DISCUSSION TITLE
Eyes dlib🙏🏻

### DISCUSSION THREAD

**Discussion by 4lt3r3go** [Bing-su/adetailer#77] [Ideas]
Greetings! and thank you for this amazing must-have extention

it would be definitive by adding " **eyes dlib** " , to fix eyes only,
like this script is doing, but i'm having issues using it 
https://github.com/ilian6806/stable-diffusion-webui-eyemask

also, if possible, a way to fix the body without touching the face (invert mask maybe would be a solution so one can choose to fix face but select invert mask to fix everything except face).. i tryed person yolo but is touching the face too, maybe i'm dumb i dunno lol

**Comment by Bing-su** [comment]
Sorry, I don't want to add any package that require to build the wheel.

**Reply by andupotorac** [reply]
> Sorry, I don't want to add any package that require to build the wheel.

Would be great to add support for fixing the eyes though. Think about After Detailer like the extension we use with ControlNet to fix the things that are always broken - be it eyes (like blank iris), face, or hands, also legs.

**Reply by DA-Charlie** [reply]
you can use specific models for different parts. I've got models for eyes on civitai

**Reply by Woisek** [reply]
Mind to share a link ... ?

**Reply by DA-Charlie** [reply]
yeah sorry..
https://civitai.com/search/models?sortBy=models_v8&query=adetailer%20eye

but personally i use these two, they work pretty well:
https://civitai.com/models/150925/eyes-detection-adetailer
https://civitai.com/models/178518/eyeful-or-robust-eye-detection-for-adetailer
